addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/schema/create-or-update-env-type-config.js

```diff
@@ -55,7 +56,7 @@ const schema = {
         {
           type: 'string',
           maxLength: 100,
-          pattern: '^[A-Za-z0-9-_ ]+$',
+          pattern: nameRegex,
```

Comment:
non-block: test this validation in create

Reply:
added to CHAM-211